.storybook/config.js

```diff
@@ -44,11 +45,29 @@ const GlobalStyle = createGlobalStyle`
   }
 `;
 
-addDecorator(story => (
+export const canonicalStyles = story => (
```

Comment:
TODO- 

This needs to either be a decorator thats always on, or one that is applied to every single story.

This current behaviour means that only psammead-image has the canonical styles, like reith

Reply:
Just in case anybody else asks the same questions I did: it looks like it's fine applying both this & the AMP decorator, as that seems to mirror our current usage in Simorgh -- it seems we always wrap the content in GlobalStyle, even if rendering for AMP